src/ngraph_mark_for_clustering.cc

```diff
@@ -47,6 +48,10 @@ namespace ngraph_bridge {
 
 using ConfirmationFunction = std::function<Status(Node*, bool*)>;
 
+#if (TF_VERSION_GEQ_1_11)
+std::unique_ptr<DeadnessAnalysis> deadness;
```

Comment:
Consider renaming this to something like `s_DeadnessAnalyzer` and making this a function static (see below)